src/lib/svg_text_utils.js

```diff
@@ -153,27 +153,16 @@ function cleanEscapesForTex(s) {
 
 function texToSVG(_texString, _config, _callback) {
     var originalRenderer,
-        originalConfig,
         originalProcessSectionDelay,
         tmpDiv;
 
     MathJax.Hub.Queue(
     function() {
-        originalConfig = Lib.extendDeepAll({}, MathJax.Hub.config);
-
         originalProcessSectionDelay = MathJax.Hub.processSectionDelay;
         if(MathJax.Hub.processSectionDelay !== undefined) {
             // MathJax 2.5+
             MathJax.Hub.processSectionDelay = 0;
         }
-
-        return MathJax.Hub.Config({
-            messageStyle: 'none',
-            tex2jax: {
-                inlineMath: [['$', '$'], ['\\(', '\\)']]
-            },
-            displayAlign: 'left',
-        });
```

Comment:
I think we still want all of the code you removed from `svg_text_utils` - store the existing config, set our own, and replace the original config at the end. That way we get the config we want without interfering with anything the rest of the page might be doing.